xonsh/xonfig.py

```diff
@@ -537,6 +550,9 @@ def _info(ns):
         pass
     data.extend([("on jupyter", jup_ksm is not None), ("jupyter kernel", jup_kernel)])
 
+    xontribs = [s.split(".")[1] for s in sys.modules if s.startswith("xontrib.")]
```

Comment:
I think there as API for getting the xontrib list. That would be good to use here

Reply:
Added + test.